src/lib/src/common/markers/marker.state.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { ElementState } from '../element-state';
+import { MarkerState } from './marker.state';
+
+describe('MarkerState', () => {
+
+  let sampleMarkerState: MarkerState;
+
+  beforeEach(() => {
+    sampleMarkerState = {
+      condition: (marker) => marker.testStatus === ElementState.Running,
+      cssClasses: 'fa-spinner fa-spin',
+      label: (marker) => `Test ${marker.name} is running`,
+    };
+  });
+
+  it('conditions can be invoked', () => {
+    // given
+    const marker = { testStatus: ElementState.Running };
+
+    // when
+    const actualResult = sampleMarkerState.condition(marker);
+
+    // then
+    expect(actualResult).toBeTruthy();
+  });
+
+  it('returns the correct label', () => {
+    // given
+    const markerState: MarkerState = {
+      condition: (marker) => marker['testStatus'] === ElementState.Running,
```

Comment:
isn't this condition misleading? I can't see that this function is called anywhere. Its probably only there to satisfy the type of MarkerState, right? Same for cssClasses.
How about
  { condition: (marker) => true, cssClasses: 'test-irrelevant-class', label ... }

Reply:
Yeah, the whole test seems a bit redundant, as it basically only tests if the function can be invoked when referenced from a surrounding object of class MarkerState…
I'll also set `cssClasses` to an empty string, as it is just as irrelevant to the test as `condition` is.